src/components/common/course-enrollments/CourseSection.jsx

```diff
@@ -25,39 +25,28 @@ class CourseSection extends React.Component {
     return <h2>{sectionTitle}</h2>;
   };
 
-  getCourseRunProps = (courseRun) => {
-    const {
-      courseRunId,
-      courseRunStatus,
-      title,
-      microMastersTitle,
-      hasEmailsEnabled,
-      linkToCourse,
-    } = courseRun;
-    const defaultCourseRunProps = {
-      courseRunId,
-      title,
-      microMastersTitle,
-      linkToCourse,
-      hasEmailsEnabled,
-    };
+  getCourseRunProps = ({
+    linkToCertificate,
+    notifications,
+    courseRunStatus,
+    ...rest
+  }) => {
     const courseRunProps = {};
     switch (courseRunStatus) {
       case 'in-progress':
-        courseRunProps.endDate = courseRun.endDate;
-        courseRunProps.linkToCertificate = courseRun.linkToCertificate;
-        courseRunProps.notifications = courseRun.notifications;
-        break;
-      case 'upcoming':
-        courseRunProps.startDate = courseRun.startDate;
+        courseRunProps.linkToCertificate = linkToCertificate;
```

Comment:
I guess I don't know why we have a certificate link showing for an in-progress course, but this might be a Product question

Reply:
There is a note about this in the Zeplin mocks. For when you get a certificate but still taking the course (supposedly that is a use case), we want to tell them they have the cert but deemphasized so they still go back to the course.